src/components/ButtonSystem/BaseButton.style.ts

```diff
@@ -0,0 +1,12 @@
+import MuiButton from '@material-ui/core/Button';
+import styled from 'styled-components';
+
+export const BaseMuiButton = styled(MuiButton).attrs(props => ({
+  disableRipple: true,
+  disableFocusRipple: true,
+}))`
+  &.Mui-focusVisible {
+    outline: blue auto 1px;
+    outline: -webkit-focus-ring-color auto 1px;
```

Comment:
lets discuss focus state in today's meeting with josh. as you pointed out, the blue focus outline isnt visible on the filled button